packages/frontend/amp/components/TopMetaExtras.tsx

```diff
@@ -120,7 +127,7 @@ export const TopMetaExtras: React.FC<{
         <TwitterHandle handle={twitterHandle} />
         <WebPublicationDate date={webPublicationDate} />
 
-        <div className={borders}>
+        <div className={composeLabsCSS(pillar, borders, bordersLabs)}>
```

Comment:
Ditto my comment above about defining a pillar neutral colour and then leveraging pillars for this.